src/views/marketplace/Extension.js

```diff
@@ -80,6 +85,42 @@ const YoutubeEmbed = ({ youtubeHash }) => {
   );
 };
 
+const InstallButton = ({ data, theme }) => {
+  if (data.app_zuid) {
```

Comment:
make this chain of IF statement an if else, then you dont need extra clauses. There will only ever be one component to render